main.tf

```
@@ -1,22 +1,38 @@
 data "aws_caller_identity" "current" {
 }
 
+data "aws_region" "current" {
+}
+
 locals {
-  account_id = data.aws_caller_identity.current.account_id
+  account_id  = data.aws_caller_identity.current.account_id
+  bucket_name = var.bucket_name == null ? "${local.account_id}-${local.region}-${var.bucket_suffix}" : var.bucket_name
+  region      = data.aws_region.current.name
```

I thought we prefered this syntax?

```suggestion
  account_id  = data.aws_caller_identity.current.account_id
  region      = data.aws_region.current.name
  
  bucket_name = coalesce(
    var.bucket_name,
    "${local.account_id}-${local.region}-${var.bucket_suffix}"
  )
```